### PR TITLE
Update security client fat timeout issue

### PR DESCRIPTION
**Pull request by adriankalafut**
Fix RTC 296796
Update the repeat rule for EE levels for com.ibm.ws.security.client_fat.

### CODE REVIEW

**Comment by adriankalafut** [comment]
#build

File: dev/com.ibm.ws.security.client_fat/fat/src/com/ibm/ws/security/client/fat/FATSuite.java [hunk 1]
```diff
@@ -43,7 +41,5 @@ public class FATSuite {
      * Run EE10 tests in LITE mode and run all tests in FULL mode.
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.security.client_fat/fat/src/com/ibm/ws/security/client/fat/FATSuite.java]:
```suggestion
     * On Windows, always run the default/empty/EE7/EE8 tests.
     * On other Platforms:
     * - if Java 8, run default/empty/EE7/EE8 tests.
     * - All other Java versions
     * -- If LITE mode, run EE9
     * -- If FULL mode, run EE10
```

**Review by Zech-Hein — CHANGES_REQUESTED** [review]

**Comment by adriankalafut** [comment]
#libby

**Comment by adriankalafut** [comment]
#build

**Review by Zech-Hein — DISMISSED** [review]
LGTM 👍

**Comment by adriankalafut** [comment]
#libby

**Review by Zech-Hein — APPROVED** [review]

**Comment by adriankalafut** [comment]
#libby

**Pull request merged**